lib/order.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :order_placed, :balance
+
+  def initialize
+    @order_placed = []
+    @balance = 0
+  end
+
+  def add(request)
+    fail 'That item is not available' if check_menu(request) == false
+
+    @order_placed << request
+    add_to_balance(request)
+  end
+
+  def check_menu(request)
+    menu = Menu.new
```

Comment:
Instead of instantiating new Menu class each time, could try initialise it and call on methods stored inside the Menu class i.e. check_menu moved into Menu class